DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs

```diff
@@ -207,5 +210,28 @@ private void AddClaimsToCookie(Claim[] claims)
                 GetRequestContext.SetCookie(claim.Type, claim.Value);
             }
         }
+
+        IEnumerable<CollaboratorWithUserIdModel> AddUserIdToCollaboratorIfExists(
```

Comment:
Yes, it looks much better as an extension method after previous changes I've made according to your comments. Moved logic to an extension method and commited.